app/src/main/index.js

```diff
@@ -0,0 +1,62 @@
+'use strict'
+
+import { app, BrowserWindow } from 'electron'
+import * as path from 'path'
+import { format as formatUrl } from 'url'
+
+const isDevelopment = process.env.NODE_ENV !== 'production'
+
+// global reference to mainWindow (necessary to prevent window from being garbage collected)
+let mainWindow
+
+function createMainWindow() {
```

Comment:
btw I know that JS is cammel-case lang, but in other js files we use rust naming convention. What we do about it?

Reply:
I fixed it to snake-case